universal-application-tool-0.0.1/app/views/questiontypes/RepeaterQuestionRenderer.java

```diff
@@ -20,10 +32,14 @@ public RepeaterQuestionRenderer(ApplicantQuestion question) {
 
   @Override
   public Tag render() {
+    ContainerTag repeaterFields = div().withId(REPEATER_FIELDS_ID);
+    // TODO: add each answer as a repeaterField
+    repeaterFields.with(repeaterField("placeholder", Optional.empty(), 0));
+
     return div()
-        .withId(question.getPath().path())
         .withClasses(Styles.MX_AUTO, Styles.W_MAX)
         .with(
+            repeaterFieldTemplate(),
```

Comment:
I think adding the template here will cause it to be submitted along with the form, which I'm not sure is what we want (it will bind something like `entity[]: ""`). I think we need to put it somewhere on the page for a block, but outside the block form - @bion has a better understanding of this

Reply:
Ahh yeah there is no "form" here. It might make sense to render repeater blocks in a very special way, that is not like how we render the other blocks, i.e. the form to be part of the repeater renderer since it doesn't have to be generically handled like blocks with other questions.

Reply:
I think I may  be missing some context on how repeated blocks can be handled differently. For now this seems ok, though it will appear in the form from `ApplicantBlockEditView`

Reply:
There's gonna be special cases for handling enumeration questions and enumeration blocks. So I was assuming the `ApplicantProgramBlockEditView` would do things a bit differently. But maybe not.

I'm gonna make a change to not use a template and just copy the last element in the `repeater_fields` with all the right things reset.